### ISSUE TITLE
Initial performance very poor after training on P3M dataset.

### ISSUE THREAD

**Issue by yashsandansing** [ZHKKKe/MODNet#202]
I have tried training MODNet on the publicly available 30k-image dataset and the performance was poor due to the unclean dataset. Now, I've changed the dataset and chosen the [P3M-10k](https://drive.google.com/uc?export=download&id=1LqUU7BZeiq8I3i5KxApdOJ2haXm-cEv1) dataset that uses 10k images with good-quality of segmentation. Now I'm fine-tuning the model using this [code ](https://colab.research.google.com/drive/1Dkg-4dZYJJji5WPScY0PdbUwtwtm_Fog?usp=sharing) where I'm training on top of the existing `modnet_photographic_portrait_matting.ckpt` with `backbone_pretrained = True`.

My losses are however drastically high. I've seen losses to be as low as [this ](https://github.com/ZHKKKe/MODNet/issues/135) but my losses are:
Semantic Loss: 9.234,
Detail Loss: 0.334,
Matte Loss: 6.392,

and seem to roll around in the same vicinity. Could you please check the code and see if there's any mistake in my data preparation/augmentation or is my training code somewhat wildly incorrect?
